client/src/pages/Home.js

```diff
@@ -31,11 +33,22 @@ const Home = () => {
     );
 
   if (!userBillTypes?.length) return <Loader />;
+
+  const types = userMonthlyBills.map((bill) => bill.name);
+  const amounts = userMonthlyBills.map((bill) => bill.amount);
+  const userBills = userMonthlyBills
+    .map((bill) => bill.amount)
+    .reduce((a, b) => a + b);
```

Comment:
this should start with a default value, like 0
```suggestion
    .reduce((a, b) => a + b, 0);
```